packages/core/package.json

```diff
@@ -57,6 +57,6 @@
     "@react-spring/types": "~9.7.5"
   },
   "peerDependencies": {
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
+    "react": "^19"
```

Comment:
Why isn't this backwards compatible? I think a lot of other libs have been able to keep that 🤔

Reply:
@joshuaellis  Yes good catch. I was confused about the breaking change brought by the new SpringContext api.
To confirm, useContext is well supported in React 16.8?